lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -382,6 +382,15 @@ def streamed_ndjson_get(url, headers)
           block_response: response_block
         )
 
+        if ['301', '302', '303', '307'].include?(response.code) && response[:location].present?
```

Comment:
Do we care if they redirect to a redirect?  @Jammjammjamm came up with this pseudocode:

```
response = RestClient::Request.execute(...)

while response_is_a_redirect do
  # should also probably specify a maximum
  # redirect count
  response = RestClient::Request.execute(...)
end
```

Reply:
set maximum redirect count to 10?